cdc/http_handler.go

```diff
@@ -260,6 +280,100 @@ func (s *Server) handleChangefeedQuery(w http.ResponseWriter, req *http.Request)
 	writeData(w, resp)
 }
 
+// handleChangefeeds dispatch the request to the specified handleFunc according to the request method.
+func (s *Server) handleChangefeeds(w http.ResponseWriter, req *http.Request) {
+	if req.Method == http.MethodGet {
```

Comment:
Can you look into https://github.com/gin-gonic/gin or other similar framework and discuss ways to keep boilerplate code to a minimum, once our API's complexity starts growing in subsequent releases of TiCDC?